modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/TagEngageWorkflowOperationHandler.java

```diff
@@ -134,7 +134,8 @@ public WorkflowOperationResult start(WorkflowInstance instance, JobContext conte
     try {
       mediaPackageForSearch = searchService.get(mediaPackageId);
     } catch (NotFoundException | UnauthorizedException e) {
-      throw new WorkflowOperationException(e);
+      logger.info("Media package {} is not published to Engage, skipping", mediaPackageId);
```

Comment:
Just realized that we should only skip on NotFound, and fail on Unauthorized. Sorry... :sweat_smile: